src/scripts/settings.js

```diff
@@ -32,6 +32,14 @@ document.querySelector('#close-review-prompt').addEventListener('click', dismiss
   link.addEventListener('click', changeActiveTab);
 });
 
+const isNotTogglApp = (val) => {
```

Comment:
If it's not too much of a hassle, can you please name this parameter to more precisely signify what is coming into the function? `val` is too generic... Is it an `url`?, a `domain`? Find something that is semantically correct.